biojava-modfinder/src/main/java/org/biojava/nbio/phosphosite/Site.java

```diff
@@ -56,25 +56,25 @@ public Site(){
 
 		int proteinIndex = -1;
 		int uniprotIndex = -1;
-		int modTypeIndex = -1;
 		int residueIndex = -1;
 		int orgIndex     = -1;
 		int groupIndex   = -1;
+		int geneIndex    = -1;
 
 		boolean inHeader = true;
 
 
 		while ((line = buf.readLine()) != null){
-			if ( line.startsWith("PROTEIN")) {
+			if ( line.startsWith("GENE")) {
```

Comment:
About your other comment: I don't think there are user generated phosphosite files. The file format is very simple and it is only used for distribution as far as I know.